pathways/inspections.py

```diff
@@ -80,12 +80,12 @@ def sample_proportion(config, shipment):
     ratio = config["inspection"]["proportion"]["value"]
     num_stems = shipment.num_stems
     num_boxes = shipment.num_boxes
-    min_boxes = config.get("min_boxes", 1)
+    min_boxes = config["inspection"]["min_boxes"]
```

Comment:
I made this change because it was defaulting to min_boxes=1 no matter what it was set to in the config. I assumed it was something with the .get method, but I wasn't sure. So I made this switch and moved min_boxes so its only specified once and directly under "inspection" (instead of being under the sample method parameters). So this change was purely to get it working (I wanted to allow 0 boxes to demonstrate rounding down), but I think specifying min_boxes right under "inspection" makes sense. Thoughts?

Reply:
My guess is that the reason you always got default was `min_boxes` being on a wrong place in config (or here taken from a wrong place). This is how the get method works (instated of raising a KeyError it returns the default/fallback):

```
>>> d = dict(a=1)
>>> d.get("a")
1
>>> d.get("a", 2)
1
>>> d.get("b", 2)
2
```

I agree with min_boxes right under "inspection". It applies to all/most of the methods and the sampling method parameters can be more pure.